solution_architecture.md

```diff
@@ -29,7 +29,7 @@ Assuming a close association of a mobile phone and its user, we equate the devic
 
 ## INTRODUCTION
 
-To reduce the spread of [COVID-19](https://www.ecdc.europa.eu/en/covid-19-pandemic) it is necessary to inform people about their close proximity to positively tested individuals. So far, health departments and citizens identified these in personal dialogues based on people’s memory, which led to a high number of unknown connections, e.g. when using public transport. 
+To reduce the spread of [COVID-19](https://www.ecdc.europa.eu/en/covid-19-pandemic) it is necessary to inform people about their close proximity to positively tested individuals. So far, health departments and persons identified these in personal dialogues based on people’s memory, which led to a high number of unknown connections, e.g. when using public transport. 
```

Comment:
I think "persons" does not work in this contact. How about "affected individuals"?

Reply:
Yes!